### PR TITLE
Adding sbox panorama to app proxy config, including logo

### PR DESCRIPTION
**Pull request by MartyFox**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/DTSPO-15781

### Change description ###
Adding sbox panorama to app proxy config, including logo

### CODE REVIEW

File: apps.yaml [hunk 1]
```diff
@@ -262,4 +262,40 @@ apps:
     tls:
       key_vault_name: "acmedtscftsboxintsvc"
       name: "wildcard-hmcts-net"
+    internalUrl: "https://backstage-sandbox.hmcts.net/"
```
Comment (reespozzi) [inline apps.yaml:265]: This doesn't seem right - there's an addition of backstage here but it's still defined below - and nothing for the panorama entry?

File: apps.yaml [hunk 2]
```diff
@@ -262,4 +262,40 @@ apps:
...
+    logoUrl: https://raw.githubusercontent.com/hmcts/azure-app-proxy/main/logos/paloalto.png
+    userAssignmentRequired: false
+    appRoleAssignments:
+      - DTS Platform Operations
+      - dcd_team_secops_v2
+      - DTS Firewall Readers
+    graphApiPermissions:
+      - "User.Read.All"
+      - "Group.Read.All"
+      - "People.Read.All"
```
Comment (reespozzi) [inline apps.yaml:278]: I don't think these are needed, app reg currently doesn't seem to have any set up?

File: apps.yaml [hunk 3]
```diff
@@ -262,4 +262,36 @@ apps:
...
+        value: adminrole
+        id: 86fee68e-ec7d-4a5a-86e4-192dc84589da
+        groups:
+          - "DTS Platform Operations"
+      - displayName: msiam_access
+        description: msiam_access
+        value: ""
+        id: 903bcba3-9c9e-4440-8b63-57ac9bd64ce3
+        groups:
+          - ""
```
Comment (reespozzi) [inline apps.yaml:294]: This won't work here - readme probably needs updating (my bad) but a group must be added -- you could add `DTS Platform Operations` or just remove this app role - it doesn't seem assigned to any groups on the existing app reg. anyway

Comment (MartyFox) [reply apps.yaml]: Fair enough, I can see this group across other app reg so wondering if its automated from somewhere else such as policies.

Comment (reespozzi) [reply apps.yaml]: I think it's automated when the app is first created, similar with a `User` role which is actually used I think

Comment (reespozzi) [reply apps.yaml]: Not sure where it comes from exactly though

**Review by reespozzi — APPROVED** [review]
Looks ok to me - won't work until DNS side sorted, but can test app permissions show correctly etc in the meantime

**Pull request merged**